README.md

```diff
@@ -563,3 +563,25 @@ To access the data, please use the [Azure Portal](https://portal.azure.com).
 ### Source Code Structure
 
 Where appropriate the source code must be split out into a hierarchy by appeal type. Several examples of this may be found under cypress tests (i.e. /packages/e2e-tests/cypress/integration), forms-web-app (i.e. /packages/forms-web-app/src/routes/), and many others.
+
+## Feature Flag
+
+We have incorporated Azure feature flag functionality into the common package for use across the solution. Feature flags should be set up in portal under the 'app configuration' section, where you can select your desired environment and configure flags accordingly, and can be used by importing the `isFeatureActive` function from `@pins/common`, i.e; 
```

Comment:
Should we not be setting up new feature flags via the terraform scripts rather than directly through Azure portal? I'd have thought if we set them up through portal, we'll have sync problems when deploying updates to the infrastructure or spinning up a failover environment in another region. 
I'd say it's fine to change the values through Azure portal when we're looking to roll out a feature (to avoid tying feature rollout to codebase deployments) though the terraform scripts should be updated to reflect the change.